pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -1,17 +1,88 @@
-{ stdenv, fetchurl, buildPythonPackage, pythonOlder,
-  cudaSupport ? false, cudatoolkit ? null, cudnn ? null,
+{ stdenv, fetchurl, fetchgit, buildPythonPackage, python, pythonOlder,
+  cudaSupport ? false, cudatoolkit ? null, cudnn ? null, nccl ? null, magma ? null,
+  mklSupport ? false, mkl ? null,
+  openMPISupport ? false, openmpi ? null,
+  buildNamedTensor ? false,
+  buildBinaries ? false,
+  cudaArchList ? null,
   fetchFromGitHub, lib, numpy, pyyaml, cffi, typing, cmake, hypothesis, numactl,
   linkFarm, symlinkJoin,
-  utillinux, which }:
+
+  # ninja (https://ninja-build.org) must be available to run C++ extensions tests,
+  ninja,
+
+  # dependencies for torch.utils.tensorboard
+  tensorboardSupport ? true, pillow, six, future, tensorflow-tensorboard,
```

Comment:
I suppose tensorflow is missing here?

```
error: undefined variable 'tensorflow' at /home/jethro/nix-config/nixpkgs/pkgs/development/python-modules/pytorch/default.nix:173:83
```

Reply:
Oops! tensorflow should have been removed from [line 173](https://github.com/NixOS/nixpkgs/pull/65041/files#diff-84b29016f8c93846ea69f3885d709770R173).